src/common/snippets/docs/debug_capabilities/parameters_dump.md

```diff
@@ -0,0 +1,26 @@
+# Snippet parameters dump
+
+Snippet parameters can be captured during the pass flow, which can be useful for debugging and optimization purposes.
+
+To turn on snippet parameters dump feature, the following environment variable should be used:
+```sh
+    OV_SNIPPETS_DUMP_BRGEMM_PARAMS="path=<path_to_csv_dump_file>" binary ...
+```
+
+Examples:
+```sh
+    OV_SNIPPETS_DUMP_BRGEMM_PARAMS="path=brgemm.csv" binary ...
+```
+
+Output example:
+```csv
+name,subgraph_name,in_type,out_type,in_shapes,out_shapes,M,N,K,m_block,n_block,k_block,acc_max_time,avg_max_time
+MatMul_437,FakeQuantize_456,i8;i8;f32,i32,1 16 128 64;1 16 64 128;1 16 64 128,1 16 128 128,128,128,64,32,-2,-2,37786,4723
+MatMul_451,FakeQuantize_456,u8;i8,i32,1 16 128 128;1 16 128 64,1 16 128 64,128,64,128,32,-2,-2,18186,2273
+MatMul_6069,FakeQuantize_6088,i8;i8;f32,i32,2 6 68 92;2 6 92 68;2 6 92 68,2 6 68 68,68,68,92,32,-2,-2,4433,2355
+MatMul_6083,FakeQuantize_6088,u8;i8,i32,2 6 68 68;2 6 68 92,2 6 68 92,68,92,68,32,-2,-2,1549,1549
+MatMul_12488,FakeQuantize_12507,i8;i8;f32,i32,? ? ? 100;? ? 100 128;? ? 100 128,? ? ? 128,-1,128,100,32,-2,-2,5823,5823
+MatMul_12502,FakeQuantize_12507,u8;i8,i32,? ? ? 128;? ? 128 100,? ? ? 100,-1,100,128,32,-2,-2,4170,1107
+MatMul_18789,FakeQuantize_18808,i8;i8;f32,i32,? ? ? ?;? ? ? ?;? ? ? ?,? ? ? ?,-1,-1,-1,32,-2,-2,4773,2757
+MatMul_18803,FakeQuantize_18808,u8;i8,i32,? ? ? ?;? ? ? ?,? ? ? ?,-1,-1,-1,32,-2,-2,3901,1300
```

Comment:
I would suggest reducing number of rows (e.g. leave only 2 ones), but organize an example as [markdown table](https://www.codecademy.com/resources/docs/markdown/tables)
P.S. this can be done at the final review stage (to avoid redundant refactoring)

Reply:
Done